NEWS.md

```diff
@@ -3,6 +3,17 @@
 ## Unversioned
```

Comment:
I would add a point at the end of each list point, similar to the other versions.